### ISSUE TITLE
Merge source.csv and fields.csv

### ISSUE THREAD

**Issue by cmgiven** [DCActionforChildren/dcaction#168]
These files contain the same rows and should be able to be merged.

**Comment by ajschumacher** [comment]
@cmgiven this is front-end? Is that because front-end calls it? Hmm... How much checking needs to be done for this? Maybe I'll take a look at it tonight (I'm at Beta NYC tonight).

**Comment by ajschumacher** [comment]
ugh I got so distracted by https://github.com/flatsheet/editdata.org and then I went and looked and it seemed like this was not a five-minute task to really figure out and do safely. Sorry! I see you committing more cool things - get 'em, @cmgiven!

**Comment by cmgiven** [comment]
Yup, the smooshing together of the two CSVs is the easy part. Just need to modify the front-end code to get everything it needs from one file instead of two.

—
